.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\Users\\Ben Bistline\\AppData\\Local\\Programs\\Python\\Python39\\python.exe"
```

Comment:
This should not be checked in, we should probably add them to .gitignore

Reply:
Yeah, that certainly shouldn't be.